README.md

```diff
@@ -6,7 +6,7 @@ Core formulae for the Homebrew package manager.
 
 ## How do I install these formulae?
 
-Just `brew install <formula>`. This is the default tap for Homebrew and is installed by default.
```

Comment:
It still is, it just uses the API.

Reply:
Why didn’t `brew tap` show the tap?

Reply:
Oh. Do you mean that it’s still “the default tap” but it’s not “installed by default”? Not sure what the difference is. If that’s the case, let me know and I can amend the commit.

Reply:
`homebrew-core` is the default tap. The formulae in `homebrew-core` are served over the API now unless you specify otherwise. This means most users no longer need local copies of the `homebrew-core` repo, which is why it is not shown by `brew tap`.

Reply:
Oh I see. Then I guess it is still “installed by default” in the sense that you can use it by default.

Thx for clarifying!